app/controllers/admin/auctions_controller.rb

```diff
@@ -35,14 +35,16 @@ def create
 
   def edit
     @auction = Admin::EditAuctionViewModel.new(Auction.find(params[:id]))
+    session[:return_to] = request.referer
```

Comment:
thoughts on extracting this into some kind of method and putting in `ApplicationController` where we already have some redirect behavior? https://github.com/18F/micropurchase/blob/develop/app/controllers/application_controller.rb#L42